project-base/app/config/wysiwyg.yml

```diff
@@ -44,14 +44,23 @@ fm_elfinder:
       editor: ckeditor
       relative_path: false
       connector:
+        debug: true
         roots:
-          uploads:
-            driver: LocalFileSystem
+          local:
+            driver: Flysystem
             path: '%shopsys.filemanager_upload_web_dir%'
-            tmb_path: '_thumbnails'
+            flysystem:
+              type: local
```

Comment:
Files form wysiwig are uploaded to local filesystem? I think they should use "abstract" filesystem from first commit.
Or maybe I don't understand these lines, if so, please explain them in the CM

Reply:
That is the way the elfinder is configured with flysystem. You can see elfinder docs for more info.

Reply:
But here it looks like the wysiwyg editor is creating its own local flysystem. So how is it?

Reply:
It is, the reason is we use OneUpFlysystemBundle and its definitions are not compatible with elfinders one...

Reply:
If wysiwyg doesn't save files into an common abstract storage, then this US isn't finished.